postgres/client.go

```diff
@@ -50,7 +50,8 @@ func (c *Client) StoreDiagnosisKeys(ctx context.Context, diagKeys []diag.Diagnos
 	}
 	defer tx.Rollback()
 
-	stmt, err := tx.PrepareContext(ctx, "INSERT INTO diagnosis_keys (key, day_number) VALUES ($1, $2)")
+	stmt, err := tx.PrepareContext(ctx, "INSERT INTO diagnosis_keys (key, day_number) VALUES ($1, $2)" +
+		"ON CONFLICT ON CONSTRAINT diagnosis_keys_pkey DO NOTHING")
```

Comment:
Maybe a raw string literal reads a bit easier here?
```suggestion
	stmt, err := tx.PrepareContext(ctx, `INSERT INTO diagnosis_keys (key, day_number) VALUES ($1, $2)
	ON CONFLICT ON CONSTRAINT diagnosis_keys_pkey DO NOTHING`)
```